pkg/util/util.go

```diff
@@ -1424,6 +1424,17 @@ func GetEnvVariable(env *[]corev1.EnvVar, name string) *corev1.EnvVar {
 	return nil
 }
 
+func GetAnnotationValue(annotations map[string]string, name string) string {
+	if annotations != nil {
+		val, err := annotations[name]
+		if err {
+			return ""
+		}
+		return val
```

Comment:
This code does the opposite of what you intended. Golang map returns `ok` as the second return value, so it is true if the value exist.
Also, if it does not exist, it should return a zero value, which in a string is `""`, so there is no need to really check for `ok`.

```suggestion
		return annotations[name]
```